docs/product/performance/cache-monitoring/index.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "Cache Monitoring"
+sidebar_order: 50
+description: "Learn more about cache monitoring with Sentry and how it can help improve your application's performance."
+---
+
+<Include name="feature-stage-alpha.mdx" />
+
+A cache can be used to speed up data retrieval, thereby improving application performance. Because instead of getting data from a potentially slow data layer, your application will be getting data from, (in a best case scenario), memory instead. Caching can speed up read-heavy workloads for applications like Q&A portals, gaming, media sharing, and social networking.
+
+A successful cache results in a high hit rate which means the data was present when fetched. A cache miss occurs when the data fetched was not present in the cache. If you have performance monitoring enabled and your application uses caching, you can see how your caches are performing with Sentry.
+
+Sentry's cache monitoring provides insights into cache utilization and latency so that you can improve the performance of the endpoints that interact with caches.
+
+Starting with the [Cache page](/orgredirect/organizations/:orgslug/performance/cache/), you get an overview of the transactions within your application that are making at least one lookup against a cache. From there, you can dig into specific cache span operations by clicking a transaction and viewing its sample list.
+
+### Prerequisites and Limitations
+
+Cache monitoring currently supports [auto instrumentation](/platform-redirect/?next=%2Fperformance%2Finstrumentation%2Fautomatic-instrumentation) for [Django's cache framework](https://docs.djangoproject.com/en/5.0/topics/cache/). Other frameworks, such as [Node.js' redis](https://www.npmjs.com/package/redis), require opt-in or custom instrumentation.
+
+### Custom and Opt-in instrumentation
```

Comment:
I think its fine to have this seperate tbh, although maybe we should change the titles to `Automatic Instrumentation` and `Custom Instrumentation`.

I don't believe `opt-in` instrumentation needs to be mentioned, this would fall under `automatic` as your opting in to automatic instrumentation.